### PR TITLE
docs: Add Conan Installation steps

### PR DESCRIPTION
**Pull request by franramirez688**
Hi there,

Conan just added the Thorvg package. This PR adds some simple steps to install it.

Thanks in advance!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +123,17 @@ Install the ThorVG package.
...
 
+### Install with Conan
+You can download and install pre-packaged ThorVG using the [Conan](https://conan.io/) package manager.
+
+Follow the instructions on [this page on how to set up Conan](https://conan.io/downloads).
+
+Install the ThorVG package:
+
+```
+conan install --requires=thorvg/0.13.0 --build=missing
```
Comment (franramirez688) [inline README.md:134]: Using the latest version:

```suggestion
conan install --requires="thorvg/[*]" --build=missing
```

**Review by mgrudzinska — APPROVED** [review]
👍🏼

**Comment by hermet** [comment]
@franramirez688 Thank you for your contribution. It would be helpful if you could share the packaging distribution guide as well: https://github.com/thorvg/thorvg/wiki/Package-Distrubution

**Comment by franramirez688** [comment]
> @franramirez688 Thank you for your contribution. It would be helpful if you could share the packaging distribution guide as well: https://github.com/thorvg/thorvg/wiki/Package-Distrubution

@hermet Indeed, good point! I just added it in my fork: https://github.com/franramirez688/thorvg/wiki/Package-Distrubution#conan
Let me know if you want me to open an issue with this link or follow another flow.
Thanks!

**Review by hermet — APPROVED** [review]

**Pull request merged**